eng/pipelines/templates/VSCode-release.template.yml

```diff
@@ -1,6 +1,6 @@
 ---
 parameters:
-  rids: ["win7-x86", "win-x64", "win10-arm64", "osx-x64", "linux-x64", "linux-arm", "linux-arm64", "linux-musl-x64" ]
+  rids: ["win-x86", "win10-arm64", "osx-x64", "linux-x64", "linux-arm", "linux-arm64", "linux-musl-x64" ]
```

Comment:
Out of curiosity: is there actually a difference between 'win7-x86' and 'win-x86'? I would have thought that would result in exactly the same bits.

Reply:
I was trying to figure out what the difference was from: https://docs.microsoft.com/en-us/dotnet/core/rid-catalog#windows-rids

But I'm guessing by marking it more specific, the runtime does some assumptions based on how specifc you get on the graph.
```
    win7-x64    win7-x86
       |   \   /    |
       |   win7     |
       |     |      |
    win-x64  |  win-x86
          \  |  /
            win
             |
            any
```

Reply:
If you diff the files you get from publishing the two, are there any differences?

Reply:
There is no difference. However, this might be an issue during the release when it copies OpenDebugAD7.exe

https://github.com/microsoft/MIEngine/blob/d1eced214596b22613b0f90f09c67a3f0c1a457e/eng/pipelines/steps/PublishOpenDebugAD7.yml#L18-L19

My thoughts are that its copying over a 64-bit exe on top the of the 32-bit one and its expecting a different runtime?

Reply:
Ah yes, that definitely seems like it is going to be a problem. I wonder if there is a project setting to control which host exe you get?

Reply:
Maybe set `AppHostRuntimeIdentifier` or just `RuntimeIdentifiers` to `win-x86` when building OpenDebugAD7 on Windows and the runtime id isn't specified?

Reply:
I'm working on a msbuild target that will sign `OpenDebugAD7.exe` since it will also be an issue for `win10-arm64`.

Reply:
Makes sense.